modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -9,6 +9,27 @@
   let(:all_claims_path) { "/services/claims/v2/veterans/#{veteran_id}/claims" }
   let(:claim_by_id_path) { "/services/claims/v2/veterans/#{veteran_id}/claims/#{claim_id}" }
   let(:scopes) { %w[claim.read] }
+  let(:profile) do
+    MPI::Responses::FindProfileResponse.new(
+      {
+        status: 'OK',
+        profile: OpenStruct.new(
+          {
+            given_names: [
+              'Abraham'
+            ],
+            family_name: 'Lincoln',
+            birth_date: '18090212',
+            ssn: '777000000',
+            icn: '1013062086V794840',
+            edipi: '1013590059',
+            participant_id: nil,
+            birls_id: '796104437'
+          }
+        )
+      }
+    )
+  end
```

Comment:
can we not use the [mvi profile](https://github.com/department-of-veterans-affairs/vets-api/blob/e70358727877c5061b7f50328d9320c8c9797b0a/spec/factories/mvi_profiles.rb#L38) factory? Not sure if it has everything but could be as simple as `build(:mpi_profile)` and then set the `profile[:status] = nil` in the test.